models/openocean/optimism/openocean_optimism_schema.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+
+models:
+  - name: openocean_v2_optimism_trades
```

Comment:
We can add roll-up spells like `openocean.trades` or `openocean_optimism.trades`, if there will be other versions of the dex.